src/model.py

```diff
@@ -1316,16 +1321,29 @@ def get_attention_inputs(self, target, expression, polarity):
         # v
         if "polar" in value:
             values = polarity
+            true_values = batch[4].permute(1,0).to(torch.device(self.device))
         elif "target" in value:
             values = target
+            true_values = batch[5].permute(1,0).to(torch.device(self.device))
         elif "expression" in value:
             values = expression
+            true_values = batch[2].permute(1,0).to(torch.device(self.device))
         else:
             values = polarity
 
+        # BUG: size problem
+        # values is of size [batch, cnn_dim, sequence]
+        # true_values is of size [batch, sequence]
+        if gold_transmission:
+            gold_influence = self.get_prob(self.current_epoch, self.find("warm_up_constant", default=5))
+            values = (  # strengthen signal for true values
+                gold_influence*true_values.bool().float().unsqueeze(-1).expand(values.shape)
+                 + (1-gold_influence)*values
+            ).detach().to(torch.device(self.device))
+            values.requires_grad = True
```

Comment:
The tokens where a label should be present are extra loud in attention at beginning, to guide model in correct direction.